infrastructure/images/k6-action/Dockerfile

```
@@ -58,9 +58,12 @@ RUN jb init && \
     jb install github.com/jsonnet-libs/k8s-libsonnet/${K8S_LIBSONNET_VERSION}@main
 
 
-ADD default_scenarios /actions/generate-k6-manifests/default_scenarios/
-ADD infra /actions/generate-k6-manifests/infra/
-ADD jsonnet /actions/generate-k6-manifests/jsonnet/
+ADD infrastructure/images/k6-action/default_scenarios /actions/generate-k6-manifests/default_scenarios/
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)